diff --git a/build/react.js b/build/react.js
index a88fb299dd40bb979b2b03de5df55f83b55f2ad8..9602b2e6b6427bb1cc2b7dd9fe557c73a567416a 100644
--- a/build/react.js
+++ b/build/react.js
@@ -1,4 +1,4 @@
-import { useMatches } from "@remix-run/react";
+import { useMatches } from "react-router";
 import * as React from "react";
 import { useTranslation } from "react-i18next";
 /**
diff --git a/build/server.d.ts b/build/server.d.ts
index 78b15659e8fc5dfaadce155ddfb55de367a914f8..b79ca613d44a7ecf192ab2667dcfacf3c4226bb7 100644
--- a/build/server.d.ts
+++ b/build/server.d.ts
@@ -1,4 +1,4 @@
-import type { Cookie, EntryContext, SessionStorage } from "@remix-run/server-runtime";
+import type { Cookie, EntryContext, SessionStorage } from "react-router";
 import { type BackendModule, type DefaultNamespace, type FlatNamespace, type InitOptions, type KeyPrefix, type Module, type Namespace, type NewableModule, type TFunction } from "i18next";
 type FallbackNs<Ns> = Ns extends undefined ? DefaultNamespace : Ns extends Namespace ? Ns : DefaultNamespace;
 export interface LanguageDetectorOption {
diff --git a/package.json b/package.json
index 2cc14cbc211e9ffade9c2186b823387a2c14d964..b7b9c3599bde2b217dee154fe2f67f9109feef4f 100644
--- a/package.json
+++ b/package.json
@@ -32,10 +32,10 @@
 		"typescript": "^5.6.2"
 	},
 	"peerDependencies": {
-		"@remix-run/cloudflare": "^2.0.0",
-		"@remix-run/deno": "^2.0.0",
-		"@remix-run/node": "^2.0.0",
-		"@remix-run/react": "^2.0.0",
+		"@react-router/cloudflare": "^7.0.0",
+		"@react-router/deno": "^7.0.0",
+		"@react-router/node": "^7.0.0",
+		"react-router": "^7.0.0",
 		"i18next": "^23.1.0",
 		"react": "^16.8.0 || ^17.0.0 || ^18.0.0",
 		"react-i18next": "^13.0.0 || ^14.0.0 || ^15.0.0"
